SalesforceFoundation/k12-architecture-kit
# [K12] Adding  All Application List View and Page Layout

# Critical Changes

# Changes
## Updated List Views and Page Layouts
- We introduced a K-12 All Applications list view on Application. If you choose to create a K-12 All Applications list view we recommend this configuration.
   - Sharing Settings: All users can see this list view
   - Filter by Owner: All attributes
   - Display order: Application Name, Applicant, Applying To, Term, Application Type, Application Status, Application Date, Application Decision Date, Applicant Decision Date, Preparer
      See [Create or Clone a List View](https://help.salesforce.com/articleView?id=customviews_lex.htm&type=5).

# Issues Closed
Closes #174
# New Metadata
Page Layout: 
- K12 Kit Application Layout 

List View: 
- K12 All Application
# Deleted Metadata

# Testing Notes
Trial Org: this should be tested only after the PR has been merged to master
Non trial org: 
- Run cci org scratch dev/devnamespaced/qa org
- Run cci flow run dev_org/dev_namespaced/qa_org
- Go to the Application tab
- Create a new Application record
- Application page layout should match the screenshot on this quip doc https://salesforce.quip.com/cXwsAPRcgr1R#daLACAf88Vy 
- Once the Application record is created, go back and check that the list view "K12 All Application" and "Recently Viewed" shows the right fields based on the screenshot and doc https://salesforce.quip.com/cXwsAPRcgr1R#daLACAlzYlb
